darts/models/forecasting/nbeats.py

```diff
@@ -30,48 +31,60 @@ class _TrendGenerator(nn.Module):
 
     def __init__(self,
                  expansion_coefficient_dim,
-                 output_dim):
+                 target_length):
         super(_TrendGenerator, self).__init__()
-        self.T = nn.Parameter(
-            torch.stack([(torch.arange(output_dim) / output_dim)**i for i in range(expansion_coefficient_dim)], 1),
-            False)
+
+        # basis is of size (expansion_coefficient_dim, target_length)
+        basis = torch.stack([(torch.arange(target_length) / target_length)**i for i in range(expansion_coefficient_dim)], 
+                            dim=1).T
+
+        self.basis = nn.Parameter(basis, requires_grad=False)
+
 
     def forward(self, x):
-        return torch.matmul(x, self.T.T)
+        return torch.matmul(x, self.basis)
 
 
 class _SeasonalityGenerator(nn.Module):
 
     def __init__(self,
-                 output_dim):
+                 target_length):
         super(_SeasonalityGenerator, self).__init__()
-        half_minus_one = int(output_dim / 2 - 1)
-        cos_vectors = [torch.cos(torch.arange(output_dim) * 2 * np.pi * i) for i in range(1, half_minus_one + 1)]
-        sin_vectors = [torch.sin(torch.arange(output_dim) * 2 * np.pi * i) for i in range(1, half_minus_one + 1)]
-        self.S = nn.Parameter(torch.stack([torch.ones(output_dim)] + cos_vectors + sin_vectors, 1),
-                              False)
+        half_minus_one = int(target_length / 2 - 1)
+        cos_vectors = [torch.cos(torch.arange(target_length) * 2 * np.pi * i) for i in range(1, half_minus_one + 1)]
+        sin_vectors = [torch.sin(torch.arange(target_length) * 2 * np.pi * i) for i in range(1, half_minus_one + 1)]
+        basis = torch.stack([torch.ones(target_length)] + cos_vectors + sin_vectors, dim=1).T
```

Comment:
could we add some comment here like 'basis is of size `(2 * int(target_length / 2 - 1) + 1, target_length)`? Similarly to the comment in the `_TrendGenerator` class.